x/pairing/keeper/grpc_query_static_providers_list.go

```diff
@@ -40,12 +42,16 @@ func (k Keeper) StaticProvidersList(goCtx context.Context, req *types.QueryStati
 	}
 
 	finalProviders := []epochstoragetypes.StakeEntry{}
-	geolocation := uint64(1)
+	var geolocationFilter pairingfilters.GeolocationFilter
+	policy := projectstypes.Policy{
+		GeolocationProfile: uint64(1),
+	}
+	_ = geolocationFilter.InitFilter(policy)
```

Comment:
should return true, check it and return error if not

Reply:
done